src/adapters/MeetingsJSONAdapter.js

```diff
@@ -204,12 +212,21 @@ export default class MeetingsJSONAdapter extends MeetingsAdapter {
       filter((event) => event.detail.ID === ID),
       tap(() => end$.next(`Meeting "${ID}" has completed.`)),
     );
-
-    const events$ = merge(audioEvents$, videoEvents$, joinEvents$, leaveEvents$, shareEvents$).pipe(
-      filter((event) => event.detail.ID === ID),
-      // Make a copy of the meeting to treat it as if were immutable
-      map((event) => ({...event.detail})),
-    );
+    const rosterEvent$ = fromEvent(document, ROSTER_CONTROL);
```

Comment:
while I prefer the singular in this case (each message emitted by this observable is a single event), let's use a plural here, for consistency with the other observables above.